src/spatial/detail/ArborX_AttachIndices.hpp

```diff
@@ -58,27 +53,18 @@ struct ArborX::AccessTraits<ArborX::Experimental::AttachIndices<Values, Index>,
   }
   KOKKOS_FUNCTION static auto get(Self const &self, int i)
```

Comment:
```suggestion
  KOKKOS_FUNCTION static auto get(Self const &self, Index i)
```
And `Index(i) -> I` below in both branches of the if constexpr

Reply:
This breaks custom index test in `tstCompileOnlyAccessTraits.cpp`.

Reply:
I would potentially want us to revisit but it does not need to be done right now